src/libLookup/Lookup.cpp

```diff
@@ -529,6 +545,10 @@ bool Lookup::GenTxnToSend(size_t num_txn,
     }
   }
 
+  if (hasTransactions) {
```

Comment:
`GenTxnToSend()`  call will happen twice, But the scenario where we pass `updateRemoteStorageDBForGenTxns = false` , `hasTransactions` will be true and `ExecuteWrite` would be called.This should not happen. Correct ?
The condition should be like this.
` if (updateRemoteStorageDBForGenTxns && hasTransactions) {
    RemoteStorageDB::GetInstance().ExecuteWrite();
  }`